### ISSUE TITLE
Consider restoring dom0 home backup to /home/subdir instead of /home

### ISSUE THREAD

**Issue by andrewdavidwong** [QubesOS/qubes-issues#2271]
Currently, when restoring a dom0 home backup, the existing contents of dom0's `/home` are moved to a subdirectory, then the contents of the backup are restored directly do dom0's `/home`.

This makes it convenient to restore a same-machine backup, since all backed up settings are automatically applied, but it makes it inconvenient to restore a cross-machine backup for the same reason (all the backed up settings are automatically applied). For example, if I create a backup on my desktop computer, which has multiple monitors and no power-saving requirements, then restore that backup on my laptop, which has a single monitor and stringent power-saving requirements, the display and power settings of the desktop get automatically applied on the laptop, and I have to manually undo/move everything. (I can't simply exclude dom0's home from the restore, because it contains some files and scripts that I need.)

**TL;DR: In general, it seems better to allow the user to have explicit control over which files and settings get applied rather than assuming that the user always wants the contents of the backup to take precedence over the user's existing files in dom0's `/home`.**

(If the concern is that restoring for backups would become more inconvenient with this change, then one possibility is to provide the user with a choice. "Which set of files do you want us to move to into a subdirectory: the existing ones, or the ones in the backup?")

**Comment by 3n7r0p1** [comment]
+1

1. backup existing /home to /home.original
2. backup restored /home to /home.restored
3. put one of these versions in /home

Recently had an issue with KDE Window Rules not being restored properly. Having the config files handy makes troubleshooting much easier than having to dig through the backup files.

**Comment by tasket** [comment]
This is a great idea. Better from a security standpoint, too.

**Comment by marmarek** [comment]
Done by @tasket here: https://github.com/QubesOS/qubes-core-admin-client/commit/30dd7acaa9deb0734f7c6b0a2bf9ff450702b94f
Dom0 home is restored into sub directory `backup-restore-<timestamp>` instead of home directly.
